TACC/Core-Portal
# 737: (FP-1678) Use "secondary" Button for UserSearchBar

## Overview: ##

Use secondary button for `<UserSearchBar>` (after secondary buttons are gray _again_ since [the newest button design]).

## Related Jira tickets: ##

* [FP-1678](https://jira.tacc.utexas.edu/browse/FP-1678)

## Summary of Changes: ##

- change button type
- make button border overlap input border, not vice versa

## Testing Steps: ##
1. Open modal of an Allocation team in which you have permission to add members.
2. Confirm that button background is gray and matches [the newest button design].

## UI Photos:

| Disabled | Normal | Hover | Active |
| - | - | - | - |
| <img width="826" alt="FP-1678 Disabled" src="https://user-images.githubusercontent.com/62723358/172956812-514c48bf-b02e-4687-9f59-f9d41ad5e4cc.png"> | <img width="826" alt="FP-1678 Normal" src="https://user-images.githubusercontent.com/62723358/172956699-b6d4aba1-9a53-4b13-9ecb-048fc7321fa0.png"> | <img width="826" alt="FP-1678 Hover" src="https://user-images.githubusercontent.com/62723358/172956703-8e55b6f9-dcdc-4eee-84f3-c65357500550.png"> | <img width="826" alt="FP-1678 Active" src="https://user-images.githubusercontent.com/62723358/172956704-561ce16f-c3b0-4213-b057-b22d3f963088.png"> |

[the newest button design]: https://github.com/TACC/Core-Portal/pull/654
